app/services/coronavirus_pages/content_builder.rb

```diff
@@ -116,4 +122,12 @@ def search_terms_in_sub_sections(items)
       [subsection[:title], labels]
     end
   end
+
+  def search_terms_in_timeline
+    return [] if github_data["timeline"].blank?
+
+    github_data["timeline"]["list"].map do |item|
+      strip_markdown(item["paragraph"])
```

Comment:
Should you be indexing the date too?

Reply:
I don't think so. Are people likely to be searching for the date? Also the date is not in a typical date format, so I think that would make it hard to search for.

Reply:
If @helengds - doesn't think we need it, I'm happy for my change request to be dismissed